dev/com.ibm.ws.security.openidconnect.client/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -244,3 +244,6 @@ tokenReuse.desc=Specifies whether JSON web tokens can be reused. Tokens must con
 
 forwardLoginParameter=Forward login parameter
 forwardLoginParameter.desc=Specifies a comma-separated list of parameter names to forward to the OpenID Connect provider. If a protected resource request includes one or more of the specified parameters, the OpenID Connect client will include those parameters and their values in the authorization endpoint request to the OpenID Connect provider.
+
+clockSkew=Clock skew
+clockSkew.desc=This is used to specify the allowed clock skew in seconds when validating the JSON web token.
```

Comment:
Change
"This is used to specify the allowed clock skew in seconds when validating the JSON web token."
To
"Specifies the allowed clock skew in seconds when you validate the JSON Web Token."

Reply:
Hi Charlotte, I'll change the sentence.  Since you want JWT to be JSON Web Token, I'll change the other instances in the file too.

Reply:
Great, thanks @barbj. Adding the ID reviewed label.